src/plugins/AccountLinkerPlugin/AccountLinker.ts

```diff
@@ -0,0 +1,21 @@
+import { inject, InjectionKey } from 'vue';
+
+export default interface AccountLinker {
```

Comment:
I think my main question is: "How to name this thing?" Better ideas welcome.

Reply:
Maybe `AuthenticationLinker`?

I think “account” isn’t the usual term in a MediaWiki context, but `UserLinker` would probably be more confusing (that would link to someone else’s user page instead).

Reply:
Good idea, thanks!